hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -312,19 +335,19 @@ public int getNumOfValidTransactions() throws IOException {
   public void addTransactions(Map<Long, List<Long>> containerBlocksMap)
       throws IOException {
     lock.lock();
-    try {
-      try(BatchOperation batch =
-          scmMetadataStore.getStore().initBatchOperation()) {
-        for (Map.Entry< Long, List< Long > > entry :
-            containerBlocksMap.entrySet()) {
-          long nextTXID = scmMetadataStore.getNextDeleteBlockTXID();
-          DeletedBlocksTransaction tx = constructNewTransaction(nextTXID,
-              entry.getKey(), entry.getValue());
-          scmMetadataStore.getDeletedBlocksTXTable().putWithBatch(batch,
-              nextTXID, tx);
-        }
-        scmMetadataStore.getStore().commitBatchOperation(batch);
+    try (BatchOperation batch = scmMetadataStore.getStore()
+        .initBatchOperation()) {
+      for (Map.Entry<Long, List<Long>> entry : containerBlocksMap.entrySet()) {
+        long nextTXID = getNextDeleteBlockTXID();
+        scmMetadataStore.getDeletedBlocksTXTable().putWithBatch(batch, nextTXID,
+            constructNewTransaction(nextTXID, entry.getKey(),
+                entry.getValue()));
       }
+      scmMetadataStore.getDeletedBlocksTXTable()
+          .putWithBatch(batch, largestTxnIdHolderKey,
+              DeletedBlocksTransaction.newBuilder().setTxID(getCurrentTXID())
+                  .setContainerID(1).setCount(1).build());
```

Comment:
@lokeshj1703 I think we need some comments. Besides can we define a new method to do the following. And because the containerID and count are dummy, we can define a const such as DUMMY_CONTAINERID = 1.
```
      scmMetadataStore.getDeletedBlocksTXTable()
          .putWithBatch(batch, largestTxnIdHolderKey,
              DeletedBlocksTransaction.newBuilder().setTxID(getCurrentTXID())
                  .setContainerID(1).setCount(1).build());
```

Reply:
Addressed in latest commit.